packages-backend/express/src/auth.ts

```diff
@@ -19,23 +19,29 @@ import { getFirstHeaderValueOrThrow } from './utils';
 type TDecodedJWT = {
   sub: string;
   iss: string;
-  [property: string]: string;
+  [property: string]: string | string[];
 };
 
 const decodedTokenKey = 'decoded_token';
 // Assign a session object to the request object.
-const writeSessionContext = <Request extends TBaseRequest>(
+export const writeSessionContext = <Request extends TBaseRequest>(
```

Comment:
Nit: to keep the exports consistent and visible we define it at the bottom of the file

```js
export { createSessionAuthVerifier };
```

Can you add this function to be exported there? Thanks